src/server/src/main/java/io/cassandrareaper/core/RepairRun.java

```diff
@@ -344,4 +350,20 @@ public RepairRun build(UUID id) {
       return new RepairRun(this, id);
     }
   }
+
+    public static void SortByRunState (List<RepairRun> repairRunCollection) {
```

Comment:
The RepairRun class is a Java Bean, so it shouldn't define such static methods indeed.
Given how the code is organized in this project, I think this method rather belongs in the RepairRunService class.

Reply:
Note that we could also define it as a [default method](https://www.baeldung.com/java-static-default-methods) in the IStorage interface.

Reply:
I'll put it in the RepairRunService, that sounds like a good idea. I can see an argument for putting it in IStorage, depends whether you see sorting as a data access consideration, or as a presentation consideration.